src/Compilers/Core/CodeAnalysisTest/Emit/EmitOptionsTests.cs

```diff
@@ -89,7 +89,8 @@ public void TestFieldsForEqualsAndGetHashCode()
                 nameof(EmitOptions.RuntimeMetadataVersion),
                 nameof(EmitOptions.TolerateErrors),
                 nameof(EmitOptions.IncludePrivateMembers),
-                nameof(EmitOptions.InstrumentationKinds));
+                nameof(EmitOptions.InstrumentationKinds),
+                nameof(EmitOptions.DefaultSourceFileEncoding));
```

Comment:
>DefaultSourceFileEncoding [](start = 35, length = 25)

missing fallback encoding here?

Reply:
You cought it before I pushed the fix :)